### PR TITLE
Support: Add seo and marketing video to help courses

### PR DESCRIPTION
**Pull request by omarjackman**
This pull request adds the youtube video containing the latest help course video for seo and marketing.

### How to test
1. Navigate to http://calypso.localhost:3000/help/courses
2. Notice that the new video appears in the "Latest course" section

### What to expect

screen shot 2017-02-09 at 2 59 11 pm

### CODE REVIEW

File: client/me/help/help-courses/course-videos.jsx [hunk 1]
```diff
@@ -19,8 +19,14 @@ export default localize( ( props ) => {
 
 	return (
 		<div className="help-courses__course-videos">
-			<Card compact className="help-courses__course-videos-label">{ translate( 'Latest course' ) }</Card>
-			{ videos.map( ( video, key ) => <CourseVideo { ...video } key={ key }/> ) }
+			<Card compact className="help-courses__course-videos-label">
+				{ translate(
+					'Latest course',
```
Comment (a8ci18n) [inline client/me/help/help-courses/course-videos.jsx]: Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Latest course' )` *ES Score: 16.82*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/help-course-videos/add-seo-video)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/me/help/help-courses/index.jsx [hunk 2]
```diff
@@ -37,6 +37,17 @@ function getCourses() {
 			],
 			videos: [
 				{
+					date: i18n.moment( new Date( 'Thur, 2 Feb 2017 17:00:00 +0000' ) ),
+					title: i18n.translate( 'Everything You Need to Know About SEO and Marketing for Your Website' ),
+					description: i18n.translate(
+						'You will leave this 60-minute session with a plan for optimizing your site for search engines ' +
+						'and an overview of basic tactics for increasing your traffic. Our Happiness Engineers will provide a' +
+						'foundation for making sure that your site meets current SEO standards. We\'ll also discuss how to ' +
```
Comment (yoavf) [inline client/me/help/help-courses/index.jsx]: Not sure why the a8ci18n bot didn't post here, but the string is missing a space between "a" and "foundation". See https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/help-course-videos/add-seo-video

Comment (omarjackman) [inline client/me/help/help-courses/index.jsx]: @yoavf Maybe thats what the "See 1 additional suggestion in the PR translation status page" in the message was referring to. Either way I didn't notice it, thanks for pointing it out.

**Comment by dllh** [comment]
This looks good to me, but we should probably also remove the "Next course" block, since now we're showing a course for Feb. 2 and then the video for the same course below. With that change, I'm cool with it and ok to :ship:

**Comment by omarjackman** [comment]
> This looks good to me, but we should probably also remove the "Next course" block, since now we're showing a course for Feb. 2 and then the video for the same course below.

@dllh you just pointed out a flaw in the data structure that drives this page. The original intention was to show a course followed by videos for that course. It's not a huge deal but It's not as easy as deleting a data row. I'll need to rethink it a bit.

**Comment by omarjackman** [comment]
The changes in the pull request were added in #11399

**Pull request closed without merging**